erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -497,7 +497,7 @@ def update_status(self, status):
 	def update_billing_status(self, update_modified=True):
 		updated_pr = [self.name]
 		for d in self.get("items"):
-			if d.purchase_invoice and d.purchase_invoice_item:
+			if getattr(d, 'purchase_invoice', None) and getattr(d, 'purchase_invoice_item', None):
```

Comment:
```suggestion
			if d.get('purchase_invoice') and d.get('purchase_invoice_item'):
```
Use `.get` instead.

This error is only occurring because the patch is getting executed before the migration of schema. Once the schema is migrated these values will be `None` unless set.

Reply:
Moving ahead with this change.